src/util/random.js

```diff
@@ -1,3 +1,5 @@
-export const randomNumber = (min, max) => {
-  return Math.random() * (max - min) + min;
+import { NINE, ZERO } from "../constants/number.js";
+
+export const generateRandomNumber = () => {
+  return Math.random() * (NINE - ZERO) + ZERO;
 };
```

Comment:
NINE, ZERO를 generateRandomNumber라는 유틸리티 함수가 의존하는 것은 예상하지 못한 상황을 줄 수 있을 것 같아요.
인자의 default value로 선언해두면 어떨까요?

```suggestion
export const generateRandomNumber = (min = ZERO, max = NINE) => {
  return Math.random() * (max - min) + min;
};
```

파라미터의 유효성까지 체크해주면 더욱 좋을 것 같구요🤭